tests/vmi_multus_test.go

```diff
@@ -1029,6 +1042,19 @@ func configInterface(vmi *v1.VirtualMachineInstance, interfaceName, interfaceAdd
 	return setInterfaceUp(vmi, interfaceName)
 }
 
+func configureSRIOVInterfaceStaticIPByMAC(virtClient kubecli.KubevirtClient, vmi *v1.VirtualMachineInstance, interfaceMac, interfaceAddress string) error {
+	if vmi == nil {
+		return fmt.Errorf("could not configure address %s for SRIOV interface with mac %s: VMI instance is nil", interfaceAddress, interfaceMac)
+	}
```

Comment:
In which case it can be nil?

Reply:
I agree that right now this method is only used in one place, and we create the vmi properly in that place before using this method. 
But maybe, in the future someone will use this method without properly creating the vm. keeps us on the safe side.
what do you say?

Reply:
We are not in the business of defensive programming. usually one expect that the retrieving a VMI also checks for errors, so I do not see this happening. But if it will, then the caller should check it IMO.

Reply:
I agree the caller should check it. but in case the caller misses it, it will receive a more accurate error message, then if it will fail further down the road.